features/calculateTotalPurchase.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Estatísticas
+
+    Scenario: Ver estatÍstica geral
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras (no tempo) são respectivamente "15000", "20000" e "10000"
+        Then os valores  "15000", "20000" e "10000" que são, respectivamente, a média, o máximo e o mínimo do valor das compras (no tempo) são mostrados
+
+    Scenario: Ver estatística por pedido
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras (por pedido) são respectivamente "150", "200" e "100"
+        When  O usuário acessa a aba de dados por pedido (ID 130)
+        Then os valores "150", "200" e "100" que são, respectivamente, a média, o máximo e o mínimo dos valores das compras em cada pedidos são mostrados
+
+    Scenario: Ver estatística dos cancelados
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras canceladas (por pedido) são respectivamente "150", "200" e "100"

```

Comment:
You need to adjust this step to be in the same format as the others that I commented before. You just need to copy the new step format and change the statistics type (in this case, "de pedidos cancelados").